package/android/src/main/java/com/mrousavy/camera/types/RecordVideoOptions.kt

```diff
@@ -0,0 +1,29 @@
+package com.mrousavy.camera.types
+
+import com.facebook.react.bridge.ReadableMap
+
+class RecordVideoOptions(map: ReadableMap) {
+  var fileType: VideoFileType = VideoFileType.MOV
+  var flash: Flash = Flash.OFF
+  var videoCodec = VideoCodec.H264
+  var videoBitRateOverride: Double? = null
+  var videoBitRateMultiplier: Double? = null
+
+  init {
+    if (map.hasKey("fileType")) {
+      fileType = VideoFileType.fromUnionValue(map.getString("fileType"))
+    }
+    if (map.hasKey("flash")) {
+      flash = Flash.fromUnionValue(map.getString("flash"))
+    }
+    if (map.hasKey("videoCodec")) {
+      videoCodec = VideoCodec.fromUnionValue(map.getString("fileType"))
```

Comment:
Is it normal that you used 'map.getString("fileType")' instead of 'map.getString("videoCoded")'? (i'm just trying to understand the changes)

Reply:
Whoops, that's a typo.
Nice spot mate, I'll fix it right away thanks!